src/pmdas/rsyslog/pmdarsyslog.pl

```diff
@@ -27,34 +27,149 @@
+#rsyslogd-pstats: @cee: { "name": "action 8", "origin": "core.action", "processed": 318, "failed": 0, "suspended": 0, "suspended.duration": 0, "resumed": 0 }
+#rsyslogd-pstats: @cee: { "name": "action 9", "origin": "core.action", "processed": 2339355, "failed": 0, "suspended": 0, "suspended.duration": 0, "resumed": 0 }
+#rsyslogd-pstats: @cee: { "name": "action 10", "origin": "core.action", "processed": 25286, "failed": 0, "suspended": 0, "suspended.duration": 0, "resumed": 0 }
+#rsyslogd-pstats: @cee: { "name": "action 11", "origin": "core.action", "processed": 3879, "failed": 0, "suspended": 0, "suspended.duration": 0, "resumed": 0 }
+#rsyslogd-pstats: @cee: { "name": "action 12", "origin": "core.action", "processed": 1170077, "failed": 0, "suspended": 0, "suspended.duration": 0, "resumed": 0 }
+#rsyslogd-pstats: @cee: { "name": "action 13", "origin": "core.action", "processed": 0, "failed": 0, "suspended": 0, "suspended.duration": 0, "resumed": 0 }
+#rsyslogd-pstats: @cee: { "name": "action 14", "origin": "core.action", "processed": 0, "failed": 0, "suspended": 0, "suspended.duration": 0, "resumed": 0 }
+#rsyslogd-pstats: @cee: { "name": "action 15", "origin": "core.action", "processed": 0, "failed": 0, "suspended": 0, "suspended.duration": 0, "resumed": 0 }
+#rsyslogd-pstats: @cee: { "name": "imudp(*:514)", "origin": "imudp", "submitted": 0 }
+#rsyslogd-pstats: @cee: { "name": "imudp(*:514)", "origin": "imudp", "submitted": 0 }
+#rsyslogd-pstats: @cee: { "name": "imptcp(*\/514\/IPv4)", "origin": "imptcp", "submitted": 1652644001, "bytes.received": 2154282781586, "bytes.decompressed": 0 }
+#rsyslogd-pstats: @cee: { "name": "imptcp(*\/514\/IPv6)", "origin": "imptcp", "submitted": 0, "bytes.received": 0, "bytes.decompressed": 0 }
+#rsyslogd-pstats: @cee: { "name": "resource-usage", "origin": "impstats", "utime": 347553851954, "stime": 88872179113, "maxrss": 415148, "minflt": 639826401, "majflt": 7039, "inblock": 1340376, "oublock": 13264688, "nvcsw": 1999618953, "nivcsw": 6535132 }
+#rsyslogd-pstats: @cee: { "name": "pstats queue", "origin": "core.queue", "size": 1, "enqueued": 18064671, "full": 0, "discarded.full": 0, "discarded.nf": 0, "maxqsize": 29 }
+#rsyslogd-pstats: @cee: { "name": "send-es-prod queue", "origin": "core.queue", "size": 2, "enqueued": 1658345033, "full": 7020, "discarded.full": 1817, "discarded.nf": 0, "maxqsize": 5000 }
+#rsyslogd-pstats: @cee: { "name": "main Q[DA]", "origin": "core.queue", "size": 0, "enqueued": 0, "full": 0, "discarded.full": 0, "discarded.nf": 0, "maxqsize": 0 }
+#rsyslogd-pstats: @cee: { "name": "main Q", "origin": "core.queue", "size": 2, "enqueued": 1676409710, "full": 0, "discarded.full": 0, "discarded.nf": 0, "maxqsize": 90022 }
+#rsyslogd-pstats: @cee: { "name": "io-work-q", "origin": "imptcp", "enqueued": 5134904, "maxqsize": 3 }
+#rsyslogd-pstats: @cee: { "name": "imudp(w0)", "origin": "imudp", "called.recvmmsg": 0, "called.recvmsg": 0, "msgs.received": 0 }
```

Comment:
And these sample outputs could form the basis of an updated version of qa/455 for regression testing the new and old formats.

Reply:
Addressed in the latest update.